site/homepage/content/getting-started/index.md

```diff
@@ -121,49 +121,53 @@ Before a device can connect to Hono and publish any data, the corresponding info
 
 ### Creating a new Tenant
 
-Choose a (random) tenant identifier and register it using the Device Registry's HTTP API (replace `my-tenant` with your identifier):
+Register a tenant it using Hono's Device Registry's management HTTP API (a random tenant identifier will be generated):
 ~~~sh
-export MY_TENANT=my-tenant
-curl -i -H "content-type: application/json" --data-binary '{"tenant-id": "'$MY_TENANT'"}' http://$REGISTRY_IP:28080/tenant
+curl -i -X POST http://$REGISTRY_IP:28080/tenants
 
 HTTP/1.1 201 Created
-Location:  /tenant/my-tenant
-Content-Length: 0
-~~~
+etag: resource_version_value
+Location:  /tenants/generated-tenant-id
```

Comment:
I assume that the response actually contains `location` instead of `Location`, doesn't it?

Reply:
correct. Fixed !